.github/workflows/ci.yml

```diff
@@ -0,0 +1,8 @@
+name: Playground Push Action
+on: [push]
+jobs:
+  Continuous-Integration:
+    runs-on: ubuntu-latest
```

Comment:
Thoughts on a smaller default like `alpine` linux

Reply:
Perhaps not relevant for the playground, but for PASS components we may want to consider the expected deployment environment and try to match that relatively closely in the CI test environment.

Reply:
This is the default Linux offered by GitHub. We will be using a self-hosted runner or a Docker image running on  a GitHub-hosted runner when up and running. This is where we will make a decision  based on what best matches the environment that each PASS component is targeted to. We should be configuring  the deployment targets and Actions runner using the same process.